priv/templates/boot.eex

```diff
@@ -44,7 +72,11 @@ REL_LIB_DIR="$RELEASE_ROOT_DIR/lib"
 ERL_OPTS="<%= release.profile.erl_opts %>"
 # When stdout is piped to a file, this is the directory those files will
 # be stored in. defaults to /log in the release root directory
-RUNNER_LOG_DIR="${RUNNER_LOG_DIR:-$RELEASE_ROOT_DIR/log}"
+
+# RELEASE_MUTABLE_DIR
+RELEASE_MUTABLE_DIR=${RELEASE_MUTABLE_DIR:-$RELEASE_ROOT_DIR/running-environment}
```

Comment:
Just 'tmp' wouldn't be a good name for a directory that contains the default `RUNNER_LOG_DIR` location.

Quoted, of course, missed that one.